mri/operators/fourier/utils.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+"""
+Common tools for MRI image reconstruction.
+"""
+
+
```

Comment:
This file just holds the generate operators from previous file. I moved the previous file to mri/operators/fourier/utils as they are all utils for fourier.
@zaccharieramzi  You ideally dont have to review this file.

Reply:
Yes, next time don't forget to use `git mv` if possible.

Reply:
I did and thats what caused this. What I needed was git copy . Which I dont think exists

Reply:
Caused what?